src/ape_socket.h

```diff
@@ -8,22 +8,32 @@
 #define __APE_SOCKET_H
 
 #include "ape_common.h"
+
 #include "ape_buffer.h"
 #include "ape_pool.h"
 #include "ape_lz4.h"
+#include "ape_dns.h"
 
 #ifdef _WIN32
-#define ioctl ioctlsocket
-#define hstrerror(x) ""
-#include "port/windows.h"
+  //#include <winsock2.h>
+  //#pragma comment(lib, "ws2_32.lib")
+  #if 0
+    #define ECONNRESET WSAECONNRESET
+    #define EINPROGRESS WSAEINPROGRESS
+    #define EALREADY WSAEALREADY
+    #define ECONNABORTED WSAECONNABORTED
```

Comment:
Yes I know, but we did not have these here before (I mean hardcoded here). And also now, it's between #if0